index.md

```diff
@@ -48,11 +30,11 @@ We're located on the 11th floor. Park in the garage and take the elevator to the
 Email: <info@paircolumbus.org>
 
 ### Friends & Links
-- @paircolumbus
+- [GitHub/paircolumbus](https://github.com/paircolumbus)
```

Comment:
`@` mentions work on organizations as well, but if you prefer making it more clear this way that's fine.